src/utils/constants.js

```diff
@@ -0,0 +1,3 @@
+export const LOGIN = 'LOGIN';
+export const LOGOUT = 'LOGOUT';
+export const PRI_COLOR = '#17a4d6';
```

Comment:
Your constants for colours should be in a different file from constants used by redux.